dockerfiles/QAT/README.md

```diff
@@ -0,0 +1,71 @@
+
+  * [Introduction](#introduction)
+  * [Building](#building)
+  * [Options](#options)
+
+## Introduction
+
+The files in this directory can be used to build a modified Kata Containers rootfs
+and kernel with modifications to support Intel QAT hardware.
+
+The generated files will need to be copied and configured into your Kata Containers
+setup.
+
+Please see the
+[Kata QAT documentation](https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md)
+for more specific QAT details.
+
+## Building
+
+The process is executed using Docker, from the directory *above* this `QAT` folder. You
+require **all** the files in this directory to build the Docker image:
+
+```sh
+$ docker build --label kataqat --tag kataqat:latest . 
+$ mkdir ./output
+$ docker run -ti --rm --privileged -v /dev:/dev -v $(pwd)/output:/output kataqat
+```
+
+> **Note:** The use of the `--privileged` and `-v /dev:/dev` arguments to the `docker run` are
+> necessary, to enable the scripts within the container to generate a roofs file system.
+
+When complete, the generated files will be placed into the output directory. Example QAT config
+files are also placed into the `config` subdirectory for references. These files may need
+modification before use, as documented in the Kata QAT
+[use-case document](https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md#copy-intel-qat-configuration-files-and-enable-virtual-functions):
```

Comment:
One of the links has a trailing section `#` link (['fragment'?](https://github.github.com/gfm/#example-509)). I don't know if you can do that by concatenating onto a [reference link](https://github.github.com/gfm/#reference-link) ?
(and, yes, I had to go look up what a 'reference link' was, as I pretty much never ever put those into markdown docs :-) ).

wdyt - if it can be done, I can add it - but, I don't know if/how to do that.... with the links being subtly different...

Reply:
Good point - let's leave as-is then.